var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -226,12 +225,16 @@ def cmake_args(self):
             ])
 
         # CUDA
-        # FIXME  -DDEAL_II_CUDA_FLAGS="-arch=sm_60"
         if '+cuda' in spec:
             options.extend([
                 '-DDEAL_II_WITH_CUDA=ON',
                 '-DDEAL_II_WITH_CXX14=OFF'
```

Comment:
Does Dealii support CUDA 9?  C++14 support was added.  If so then this could be left on when `cuda@9:`.

Reply:
good catch, it does. I will adjust.